datatypes-jws/src/commonMain/kotlin/at/asitplus/crypto/datatypes/jws/JweEncrypted.kt

```diff
@@ -22,7 +22,7 @@ data class JweEncrypted(
     val authTag: ByteArray
 ) {
     val header: JweHeader?
-        get() = JweHeader.deserialize(headerAsParsed.decodeToString())
+        get() = JweHeader.deserialize(headerAsParsed.decodeToString()).getOrNull()
```

Comment:
Now that I see it, it makes me uneasy. Would it not be better to force this deserialization on initialization s.t. the deserialization error shows up when deserializing a JweEncrypted object?

Ofc, it would be wise to check for similar constructs across the codebase